CodeEditModules/Modules/AppPreferences/src/Model/Source Control/SourceControlPreferences.swift

```diff
@@ -40,9 +40,9 @@ public extension AppPreferences {
         /// Indicates whether or not we should include the upsteam
         public var includeUpstreamChanges: Bool = false
         /// The selected value of the comparison view
-        public var comparisonView: Int = 0
+        public var comparisonView: ComparisonView = .localLeft
         /// The selected value of the control navigator
-        public var controlNavigator: Int = 0
+        public var controlNavigator: ControlNavigator = .sortName
```

Comment:
Since this is a setting which will be in `preferences.json` make sure to choose a name that is descriptive to what it does.

Reply:
What name do you like best?

Reply:
Maybe something like `controlNavigatorOrder`